src/Compilers/Core/Portable/Operations/IRaiseEventExpression.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Semantics
+{
+    /// <summary>
+    /// Represents a raise event expression.
+    /// </summary>
+    /// <remarks>
+    /// This interface is reserved for implementation by its associated APIs. We reserve the right to
+    /// change it in the future.
+    /// </remarks>
+    public interface IRaiseEventExpression : IHasArgumentsExpression
```

Comment:
>IRaiseEventExpression [](start = 21, length = 21)

I am curious why is this an expression rather than a statement?

Reply:
There is no such thing as raise event expression in VB or C#.

---
In reply to: [140924417](https://github.com/dotnet/roslyn/pull/22243#discussion_r140924417) [](ancestors = 140924417)

Reply:
That's a good point. I think it was mostly because we initially wanted this interface to represent C# raise event as well. To make this an statement, I probably have to change the name of `IHasArgumentExpression` to `IHasArgument`.

Reply:
Thoughts? @dotnet/analyzer-ioperation